src/schedule/zephyr_dma_domain.c

```diff
@@ -219,6 +231,15 @@ static int register_dma_irq(struct zephyr_dma_domain *domain,
 			crt_data->channel = crt_chan;
 			crt_data->dt = dt;
 
+			/* attach crt_data to pipeline task.
+			 *
+			 * this will be used in
+			 * zephyr_dma_domain_is_pending to decide
+			 * whether there was a DMA IRQ for the channel
+			 * associated with this task.
+			 */
```

Comment:
chars per line limit is at 100. So we can compact the compact the comments here on fewer lines.